aggregator/pom.xml

```diff
@@ -43,6 +43,10 @@
 					<groupId>org.apache.tomcat.embed</groupId>
 					<artifactId>tomcat-embed-core</artifactId>
 				</exclusion>
+				<exclusion>
```

Comment:
What does this package used for? I don't see any code use it in this PR

Reply:
This update is associated with the new come out vulnerability,  this Package is used as a library, without it the app will throw errors.